examples/mongocxx/mongodb.com/documentation_examples.cpp

```diff
@@ -1180,6 +1193,85 @@ void delete_examples(mongocxx::database db) {
     }
 }
 
+static void snapshot_examples(mongocxx::client& client) {
+    // Write concern "majority" needed to avoid the following error below:
+    //
+    // Unable to read from a snapshot due to pending collection catalog changes;
+    // please retry the operation.
+    //
+    // See also: https://jira.mongodb.org/browse/SERVER-41532
+
+    // Start Snapshot Query Example 1
```

Comment:
Suggest doing the `insert_one` operations outside of the `// Start Snapshot Query Example 1` comment to match the example in the drivers ticket.